modules/KalturaSupport/resources/mw.KPushCuePointsManager.js

```diff
@@ -0,0 +1,208 @@
+(function (mw, $) {
+	"use strict";
+	mw.KPushCuePointsManager = function (embedPlayer) {
+		return this.init(embedPlayer);
+	};
+
+	mw.KPushCuePointsManager.getInstance = function (embedPlayer) {
+		if (!embedPlayer.KPushCuePointsManager) {
+			embedPlayer.KPushCuePointsManager = new mw.KPushCuePointsManager(embedPlayer);
+		}
+		return embedPlayer.KPushCuePointsManager;
+	};
+
+	mw.KPushCuePointsManager.prototype = {
+		bindPostfix: '.KPushCuePointsManager',
+		init: function (embedPlayer) {
+			// Remove any old bindings:
+			this.destroy();
+			// Setup player ref:
+			this.listeners = {};
+			this.embedPlayer = embedPlayer;
+			this.kClient = mw.kApiGetPartnerClient(this.embedPlayer.kwidgetid);
+			this.pushServerNotification = mw.KPushServerNotification.getInstance(this.embedPlayer);
+			this.addBindings();
+			this.lastDispatchedCpTime = null;
+		},
+		addBindings: function () {
+			// bind to cue point events
+			var _this = this;
+			this.embedPlayer.bindHelper("monitorEvent onpause onplay", function (e) {
+				_this.checkAllRegisteredNotifications(e);
+			});
+		},
+		findCuePointdataByTime: function (t) {
+			for (var key in this.listeners) {
```

Comment:
what is i ask for cue point by time  near an existing cue point?  (for example when seeking with scrubber)